server/client.js

```diff
@@ -0,0 +1,13 @@
+const fetch = require("node-fetch");
+
+(async () => {
+    await fetch("http://localhost:4000/send-email", {
```

Comment:
Email will currently be sent from my account but we will change it to whatever email you wanna use